### PR TITLE
Add Oasis Sapphire chains

### PR DESCRIPTION
**Pull request by matevz**
In addition to the Oasis Emerald chains, this PR adds the new confidential Oasis Sapphire chains:
- Oasis Sapphire Testnet
- Oasis Sapphire Mainnet (upcoming)

### CODE REVIEW

**Review by ligi — COMMENTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by matevz** [comment]
> Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json) e.g. run `npx prettier --write _data/*/*.json`

Oops, thanks. Fixed.

File: _data/chains/eip155-23294.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://docs.oasis.io/dapp/sapphire",
+  "shortName": "sapphire",
+  "chainId": 23294,
+  "networkId": 23294,
+  "explorers": [
+    {
+      "name": "Oasis Sapphire Explorer",
+      "url": "https://explorer.sapphire.oasis.dev",
```
Comment (ligi) [inline _data/chains/eip155-23294.json:19]: is not reachable

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-23294.json (hunk 1 above)
Comment (matevz) [reply _data/chains/eip155-23294.json:19]: Yeah, the Mainnet deployment is scheduled for Q4 2022. Should we register the chain ID with another PR then or can this be combined in this single PR?

Comment (ligi) [reply _data/chains/eip155-23294.json:19]: you can add the explorer later and add status incubating - just cannot check the explorer EIP compatibility now

Comment (matevz) [reply _data/chains/eip155-23294.json]: Done.

**Review by ligi — APPROVED** [review]

**Comment by matevz** [comment]
Can we merge this PR or is there something else that needs to be done? (I don't have write permissions...)

**Review by ligi — APPROVED** [review]

**Pull request merged**